charts/opentelemetry-operator/values.schema.json

```diff
@@ -2108,6 +2108,105 @@
           "examples": [
             {}
           ]
+        },
+        "resources": {
```

Comment:
it seems we have a definition for resources already [here](https://github.com/open-telemetry/opentelemetry-helm-charts/blob/dea8cd4c0d460557e5126338e004d32bc99cd0b3/charts/opentelemetry-operator/values.schema.json#L671-L769), should we make this it's own schema type and re-use it?